messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json

```diff
@@ -0,0 +1,27 @@
+{
+    "account_sid": "$TWILIO_ACCOUNT_SID",
+    "date_created": "2023-06-02T14:34:25Z",
+    "date_updated": "2023-06-02T14:34:25Z",
+    "friendly_name": "WhatsApp OTP button template",
+    "language": "en",
+    "links": {
+        "approval_create": "https://content.twilio.com/v1/Content/HXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/ApprovalRequests/whatsapp",
+        "approval_fetch": "https://content.twilio.com/v1/Content/HXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/ApprovalRequests"
+    },
+    "sid": "HXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "types": {
+        "whatsapp/authentication": {
+            "actions": [
+                {
+                    "copy_code_text": "Copy verification code",
+                    "type": "COPY_CODE"
+                }
+            ],
+            "add_security_recommendation": true,
+            "body": "{{1}}",
```

Comment:
I think I'm missing this -- what is the value that this {{1}} is placeholding for?

Reply:
Great question ... I had to go look it up myself: https://www.twilio.com/docs/content/whatsappauthentication#send-wa-authentication-templates-created-with-content-templates 

This type of template just has a pre-defined variable, the value of which is passed when sending the API request to send a message with that specific template.

Reply:
Oh sorry, I wasn't seeing that variable defined in the curl sample and got confused. All good!